{
    "title": "CVS 1.12 and earlier on Debian GNU/Linux does not properly handle when a mapping for the current repository does not exist in the cvs-repouids file, which allows remote attackers to cause a denial of service (server crash).",
    "score": 5.0,
    "severity": "MEDIUM",
    "cwe_id": "NVD-CWE-Other",
    "cvss_vector": "CVSS:2.0/AV:N/AC:L/Au:N/C:N/I:N/A:P",
    "cpe_match": [
        "cpe:2.3:a:cvs:cvs:1.11.14:*:*:*:*:*:*:*",
        "cpe:2.3:a:cvs:cvs:1.10.6:*:*:*:*:*:*:*",
        "cpe:2.3:a:cvs:cvs:1.11.1:*:*:*:*:*:*:*",
        "cpe:2.3:a:cvs:cvs:1.11:*:*:*:*:*:*:*",
        "cpe:2.3:a:cvs:cvs:1.11.4:*:*:*:*:*:*:*",
        "cpe:2.3:a:cvs:cvs:1.11.16:*:*:*:*:*:*:*",
        "cpe:2.3:a:cvs:cvs:1.11.5:*:*:*:*:*:*:*",
        "cpe:2.3:a:cvs:cvs:1.10.8:*:*:*:*:*:*:*",
        "cpe:2.3:a:cvs:cvs:1.11.15:*:*:*:*:*:*:*",
        "cpe:2.3:a:cvs:cvs:1.10.7:*:*:*:*:*:*:*",
        "cpe:2.3:a:cvs:cvs:1.11.3:*:*:*:*:*:*:*",
        "cpe:2.3:a:cvs:cvs:1.11.1_p1:*:*:*:*:*:*:*",
        "cpe:2.3:a:cvs:cvs:1.11.10:*:*:*:*:*:*:*",
        "cpe:2.3:a:cvs:cvs:1.11.11:*:*:*:*:*:*:*",
        "cpe:2.3:a:cvs:cvs:1.10:*:*:*:*:*:*:*",
        "cpe:2.3:a:cvs:cvs:1.11.6:*:*:*:*:*:*:*",
        "cpe:2.3:a:cvs:cvs:1.12:*:*:*:*:*:*:*",
        "cpe:2.3:a:cvs:cvs:1.11.2:*:*:*:*:*:*:*"
    ],
    "cve_source": [
        "NVD"
    ],
    "ttpMapping": [
        {
            "techniqueId": "T1498",
            "techniqueName": "Network Denial of Service",
            "confidence": "10.0"
        },
        {
            "techniqueId": "T1499",
            "techniqueName": "Endpoint Denial of Service",
            "confidence": "10.4"
        }
    ]
}
